corehq/apps/data_analytics/south_migrations/0003_fill_user_type.py

```diff
@@ -0,0 +1,39 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+class Migration(DataMigration):
+
+    def forwards(self, orm):
+        for malt_row in orm.MALTRow.objects.all():
+            malt_row.user_type = "WebUser" if malt_row.is_web_user else "CommCareUser"
+            malt_row.save()
```

Comment:
If you're concerned about performance, you might be able to do this with a single [update](https://docs.djangoproject.com/en/1.7/ref/models/querysets/#update), rather than saving each row individually.  I think it'd be something like
```python
orm.MALTRow.objects.filter(is_web_user=True).update(user_type="WebUser")
orm.MALTRow.objects.filter(is_web_user=False).update(user_type="CommCareUser")
```